editor/src/messages/input_mapper/input_mappings.rs

```diff
@@ -401,7 +401,8 @@ pub fn input_mappings() -> Mapping {
 		entry!(KeyDown(Tab); modifiers=[Control], action_dispatch=PortfolioMessage::NextDocument),
 		entry!(KeyDown(Tab); modifiers=[Control, Shift], action_dispatch=PortfolioMessage::PrevDocument),
 		entry!(KeyDown(KeyW); modifiers=[Accel], action_dispatch=PortfolioMessage::CloseActiveDocumentWithConfirmation),
-		entry!(KeyDown(KeyO); modifiers=[Accel], action_dispatch=PortfolioMessage::OpenDocument),
+		entry!(KeyDown(KeyW); modifiers=[Accel,Alt], action_dispatch=PortfolioMessage::CloseAllDocumentsWithConfirmation),
```

Comment:
It is perhaps a bit confusing that there are entries for `PortfolioMessage::CloseAllDocumentsWithConfirmation` and `DialogMessage::CloseAllDocumentsWithConfirmation` bound to different things. Could you consider removing one of them from the list? I think the preferred shortcut is <kbd>Ctrl</kbd> + <kbd>W</kbd>.